reference/docs-conceptual/PowerShell-Support-Lifecycle.md

```diff
@@ -24,13 +23,34 @@ guarantee there that the community will address or resolve your issue in a timel
 have a problem that requires immediate attention, you should use the traditional, paid support
 options.
 
-## Lifecycle of PowerShell Core
+## LifeCycle of PowerShell 7
+
+With the release of PowerShell 7, PowerShell continues to be supported under the [Microsoft Modern Lifecycle Policy][modern],
+but support dates are linked to [.NET Core's support lifecycle][Long-Term]. In this servicing
+approach, customers can choose Long Term Support (LTS) releases or Current releases. PowerShell 7.0
+is an LTS release. Support ends with the support of .NET Core 3.1. The next LTS release follows the
+next .NET core LTS release. See the [PowerShell releases end of life table](#powershell-releases-end-of-life)
+for current ending support dates. LTS release updates only contain critical security and servicing
+updates and fixes that are designed to avoid or minimize impact to existing workloads.
+
+A Current release is a release that occurs between LTS releases. Current releases can contain
+critical fixes, innovations, and new features. A current release is supported for three months after
```

Comment:
Three or six?

Reply:
Three. That is the .Net Core policy.

Reply:
This is specifically for a "Current" release. in short, LTS releases will be supported for 3 years or 12 months after a subsequesnt LTS release, whichever is longer. 

"Current" releases will be supported 3 months after a subsequent "Current" or LTS release

Reply:
There is a problem. After .Net Core next servicing release we get PowerShell servicing release later 1-2 months. So we get EOL of the .Net Core release before the PowerShell Core release.

Reply:
The PowerShell team will still support the the versions for the documented time frames. @joeyaiello, do you have any input on this?